src/selectors/getAllProjectKeys.js

```diff
@@ -3,8 +3,15 @@ import {createSelector} from 'reselect';
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((firstProj, secondProj) => {
+      if (firstProj.updatedAt === null) {
```

Comment:
Project style is to prefer `isNull` to the explicit equality check

Reply:
Ah okay can do